deku-derive/src/lib.rs

```diff
@@ -1,45 +1,296 @@
-use darling::{ast, FromDeriveInput, FromField, FromMeta, FromVariant};
+use darling::{ast, FromDeriveInput, FromField, FromVariant};
 use proc_macro2::TokenStream;
 use quote::quote;
 mod macros;
 use crate::macros::{deku_read::emit_deku_read, deku_write::emit_deku_write};
+use syn::punctuated::Punctuated;
+use syn::spanned::Spanned;
+
+/// # Note
+/// We use this instead of `DekuReceiver::init` because handle everything in one struct is hard to use,
```

Comment:
Can remove this comment now that `DekuReceiver::init` is gone

Reply:
Yes. Actually it should be in a commit message, instead comment.